jujugui/static/gui/src/test/proxy-module.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+// This module is used as a proxy for css modules in the jest tests.
```

Comment:
Is this still required with the transform in the babelrc? 
Also, why shouldn't they be loaded?

Reply:
We don't do anything with the css that is imported so we ignore it (rather than adding a lib to handle it). You're right this is no longer required.